src/main/kotlin/org/rust/ide/navigation/goto/RsTargetElementEvaluator.kt

```diff
@@ -17,7 +28,35 @@ class RsTargetElementEvaluator : TargetElementEvaluatorEx2() {
      * Note that if this method returns null, it means
      * "use default logic", i.e. call `ref.resolve()`
      */
-    override fun getElementByReference(ref: PsiReference, flags: Int): PsiElement? = null
+    override fun getElementByReference(ref: PsiReference, flags: Int): PsiElement? {
```

Comment:
It's because `PsiReference` (or reference element) isn't passed to `getGotoDeclarationTarget` method. We need reference element to a find receiver type and a derive attribute under it.

Reply:
Okay :(